src/fire.h

```diff
@@ -2,6 +2,9 @@
 #ifndef FIRE_H
 #define FIRE_H
 
+const int MAX_FIRE_SIZE = 4;
```

Comment:
Have you tested this carefully? We currently only support field density up to 3 (`MAX_FIELD_DENSITY` is 3) and the fields data (inside of `field_t`) has only data for that many density levels.

A field density of more than 3 will lead to errors or just undefined behaviour.

Reply:
Resolved: see commit 1d72fd0